frontend/public/components/monitoring.tsx

```diff
@@ -1,5 +1,9 @@
 import * as classNames from 'classnames';
 import * as _ from 'lodash-es';
+import * as React from 'react';
+import { Helmet } from 'react-helmet';
+import { connect } from 'react-redux';
+import { Link, Redirect, Route, Switch } from 'react-router-dom';
```

Comment:
Is there a reason for moving these? They were previously in alphabetical order. Is there preferred order?

Reply:
this is the pattern that we follow in the files under dev-console package.